components/CreateForm.js

```diff
@@ -0,0 +1,127 @@
+import { useState, useRef } from "react";
+import styled from "styled-components";
+import { Button, ButtonContainer } from "./Buttons";
+
+function CreateForm() {
+  const [steps, setSteps] = useState(1);
+  const buttonRef = useRef();
+
+  function scrollToButton() {
+    setTimeout(() => {
+      buttonRef.current.scrollIntoView({ behavior: "smooth" });
+    });
+  }
+
+  function handleAddStep() {
+    setSteps((prevSteps) => prevSteps + 1);
+    scrollToButton();
+  }
+
+  return (
+    <>
+      <FormContainer>
+        <FormCard>
+          <StyledLabel isPrimary>
+            <LabelText>Tutorial title</LabelText>
+            <StyledInput
+              placeholder="e.g. Repair a faucet"
+              aria-placeholder="e.g. Repair a faucet"
+            />
+          </StyledLabel>
+        </FormCard>
+        {[...Array(steps)].map((step, index) => {
+          return (
+            <FormCard key={index.toString()}>
+              <StepNumber>Step {index + 1}</StepNumber>
+              <StyledLabel isPrimary={false}>
+                <LabelText>Step title</LabelText>
+                <StyledInput
+                  placeholder="e.g. Prepare your tools"
+                  aria-placeholder="e.g. Prepare your tools"
+                />
+              </StyledLabel>
+              <StyledLabel isPrimary={false}>
+                <LabelText>Picture URL</LabelText>
+                <StyledInput
+                  placeholder="https://www..."
+                  aria-placeholder="https://www..."
+                />
+              </StyledLabel>
+              <StyledLabel isPrimary={false}>
+                <LabelText>Step description</LabelText>
+                <StyledTextarea
+                  placeholder="Enter a description"
+                  aria-placeholder="Enter a description"
+                />
+              </StyledLabel>
+            </FormCard>
+          );
+        })}
+      </FormContainer>
+      <ButtonContainer>
+        <Button ref={buttonRef} isPrimary onClick={handleAddStep}>
```

Comment:
This buttonRef is one thing I didn't understand. Maybe you can explain it to me tomorrow. 

I have another question for my understanding, but it's tooo embarassing to write it down here, so maybe we have the opportunity to talk about it tomorrow, too, in a separate room. (I guess it will be done in three minutes.)